typed-racket-doc/typed-racket/scribblings/guide/more.scrbl

```diff
@@ -89,13 +89,21 @@ of @racket[Number]s.
 @racketblock[(lambda ([x : String] (unsyntax @tt["."]) [y : Number #,**]) (apply + y))]
 
 This function has the type @racket[(-> String Number #,** Number)].
+To specify the return type, add a type annotation after the arguments:
```

Comment:
Maybe add the word "explicitly" here, so that it's not read as "if you don't specify the return type, then it doesn't exist"?

Reply:
That reading seems unlikely. There's an example function right above with no annotation and we say it returns `Number`.